content-src/components/ActivityFeed/ActivityFeed.scss

```diff
@@ -9,8 +9,12 @@
 .feed-item {
   display: flex;
   padding-left: $feed-gutter-h + $feed-circle-size;
-  padding-bottom: $feed-gutter-v;
+  padding-right: $feed-gutter-h;
+  padding-bottom: $feed-gutter-v / 2;
+  padding-top: $feed-gutter-v / 2;
   position: relative;
+  border: 1px solid $white;
```

Comment:
I think the `$white` may be causing some mild UI issues here. You can see a slight gap in the vertical bar. Not sure if we can do a transparent border instead. Will investigate further.